cypress/e2e/helpers/surveyHelper.ts

```diff
@@ -35,61 +38,129 @@ export const verifySurveyCard = (
 
   const currentDate = new Date();
   const formattedDateText = format(currentDate, "EEEE, MMMM dd");
   cy.get(DATE_TEXT).should("contain", formattedDateText);
 
   cy.contains(surveyStrings.WHAT_IS_YOUR_WEIGHT_TODAY);
-  cy.contains(surveyStrings.PLEASE_INSERT_VALUE_BETWEEN_60_700_LBS);
+  cy.contains(surveyStrings.PLEASE_WRITE_FULL_NUMBER_DECIMAL);
   cy.contains(surveyStrings.REQUIRED);
   cy.get(VALUE_LBS_INPUT).should(
     "have.attr",
     "placeholder",
     surveyStrings.LBS_000_0,
   );
-  cy.get(LBS_TEXT).should("contain", surveyStrings.LBS);
+  cy.get(VALUE_TYPE).should("contain", surveyStrings.LBS);
+  cy.contains(surveyStrings.PLEASE_INSERT_VALUE_BETWEEN_60_700_LBS);
 
-  cy.get(PROGRESS_BAR_50_IMG).should("exist");
-  cy.get(VALUE_PROGRESS_BAR_TEXT).should("contain", surveyStrings.PERCENT50);
+  cy.get(progressBarImageSelector).should("exist");
+  cy.get(VALUE_PROGRESS_BAR_TEXT).should("contain", percentValue);
 };
 
-export const verifyWeightIntervalMsgError = (showMsgError: boolean) => {
+export const verifyIntervalMsgError = (
+  showMsgError: boolean,
+  msgError: string,
+) => {
   if (showMsgError) {
     cy.get("p.MuiFormHelperText-root.Mui-error")
       .should("be.visible")
-      .and("contain", surveyStrings.PLEASE_INSERT_VALUE_BETWEEN_60_700_LBS);
+      .and("contain", msgError);
   } else {
     cy.get("p.MuiFormHelperText-root.Mui-error").should("not.exist");
   }
 };
 
-export const verifyBodyWeightConfirmScreen = (weightInLbs: string) => {
-  cy.get(PRIMARY_BUTTON).contains(generalStrings.NEXT).click();
+export const submitSurvey = () => {
+  cy.contains("button", generalStrings.SUBMIT_SURVEY).click();
+  cy.get(SNACK_BAR_ALERT_ID)
+    .should("be.visible")
+    .contains(surveyStrings.SURVEY_COMPLETED);
+  cy.wait(7000);
+};
+
+export const verifyBodyWeightConfirmScreen = (
+  weightInLbs: string,
+  progressBarImageSelector: string,
+  percentValue: string,
+) => {
+  cy.get(CONTAINED_PRIMARY_BUTTON).contains(generalStrings.NEXT).click();
 
   cy.contains(BUTTON, surveyStrings.CLOSE).should("be.visible");
   cy.contains("h4", surveyStrings.SERISD001_BODY_WEIGHT);
   cy.contains(surveyStrings.WHAT_IS_YOUR_WEIGHT_TODAY);
   cy.contains(`${weightInLbs} ${surveyStrings.LBS}`);
   cy.contains("button", generalStrings.PREVIOUS).should("be.visible");
 
-  cy.get(PROGRESS_BAR_100_IMG).should("exist");
-  cy.get(VALUE_PROGRESS_BAR_TEXT).should("contain", surveyStrings.PERCENT100);
+  cy.get(progressBarImageSelector).should("exist");
+  cy.get(VALUE_PROGRESS_BAR_TEXT).should("contain", percentValue);
 };
 
-export const submitSurvey = () => {
-  cy.contains("button", generalStrings.SUBMIT_SURVEY).click();
-  cy.get(SNACK_BAR_ALERT_ID)
-    .should("be.visible")
-    .contains(surveyStrings.SURVEY_COMPLETED);
-  cy.wait(7000);
+export const validatetMsgOnScreen = (surveyTitle: string) => {
+  cy.contains("h5", surveyTitle).should("be.visible");
+};
+export const verifyBodyCompositionContentInPercent = (
```

Comment:
Hmm didn't get why "InPercent" on the function name
```suggestion
};

export const verifyBodyCompositionContentInPercent = (
```

Reply:
This refers to the verification of body composition in percentage.

Reply:
Got it.. but I think we should follow same pattern as the other screen and you question name to identify the function. I think on this way it gets easier to understand from which question this function is about

Reply:
Ok, I undertand, I renamed it to verifyBodyCompositionScreenContent, ok?